### DISCUSSION TITLE
Add Support to provide the buttons adjacent to the NavTabs

### DISCUSSION THREAD

**Discussion by SorexBentley** [iTwin/iTwinUI-react#806] [General]
We have started integrating iTwin components to our project. But need some more additional support as mentioned below : -

1) Need to add support for the buttons needed adjacent to the NavTabs as shown below in Image : -

![image](https://user-images.githubusercontent.com/97873094/187401334-808e684a-6308-4b3b-839f-c53f34fe039b.png)

As of now we don't any prop to Handle the adjacent button.

I think it's best to add support for onClick() button within those <NavTabs />.

**Comment by bentleyvk** [comment]
Hello @SorexBentley, we currently don't support having anything inside tabs. But for your case I think it would be enough to have some container element that has tabs and button inside:
```jsx
<div style={{ display: 'flex', justifyContent: 'space-between' }}>
  <Tabs />
  <Button />
</div>
```

**Reply by SorexBentley** [reply]
The above styles and concept does not work as you can see the scenario below : -

Suggested concept  code : -
![image](https://user-images.githubusercontent.com/97873094/190983008-d9052de3-1071-4d87-96d2-2bbbba569f2a.png)

Result : -
![image](https://user-images.githubusercontent.com/97873094/190983659-ac61d526-2141-4ca1-b5e2-3099e6121945.png)

Expected result : -
![image](https://user-images.githubusercontent.com/97873094/190983789-3d40e493-aff4-4128-9e13-f0664c38644b.png)

where the button should be on the top of  NavTab bottom border.

I think we need to provide some prop which can fix this.

**Reply by FlyersPh9** [reply]
@SorexBentley this might need a change in iTwinUI-CSS before being available in iTwinUI-React.  I've created a new issue for us to track this enhancement: https://github.com/iTwin/iTwinUI/issues/770
